src/Symfony/Component/Routing/Matcher/Dumper/StaticPrefixCollection.php

```diff
@@ -187,6 +185,11 @@ private function getCommonPrefix(string $prefix, string $anotherPrefix): array
             }
         }
         restore_error_handler();
+        if ($i < $end && 0b10 === (\ord($prefix[$i]) >> 6) && preg_match('//u', $prefix.' '.$anotherPrefix)) {
```

Comment:
I would add a comment here explaining what that snippet of code do.

Reply:
comment added